### ISSUE TITLE
response.sendRedirect doesn't work in index.jsp

### ISSUE THREAD

**Issue by hobojed** [BladeRunnerJS/brjs#1450]
Our login aspect `index.jsp` contains a call to response.sendRedirect which is intended to redirect the user to the main app under certain conditions.

This works outside of BRJS, but running the same JSP in BRJS just results in a blank page loading, instead.

Here is the code we are using that is failing:

```
if (!redirectedAtLeastOnce || loggedOut || request.getRemoteUser() != null ) {
	response.sendRedirect(request.getContextPath() + "/");
	session.setAttribute("REDIRECTED_AT_LEAST_ONCE",true);
	return;
}
```

**Comment by andyberry88** [comment]
Redirects work in `unbundled-resources` because the content plugin uses the `UrlContentAccessor.handleRequest()` method which forwards the request to the Servlet context if it's running within the BRJS dev server. The code that loads the index page uses  `UrlContentAccessor.writeLocalUrlContentsToOutputStream()` which does an 'include' instead.

So to fix this `AppRequestHandler.getIndexPageContent()` needs to be changed to use `UrlContentAccessor.handleRequest()`.

**Comment by thecapdan** [comment]
works. tested in chrome

**Comment by thecapdan** [comment]
sorry, this seems to have caused a regression. can be easily reproduced by loading the IT-app

![image](https://cloud.githubusercontent.com/assets/6085260/8501965/8dacc99c-21a2-11e5-97c7-eb6dc59448ff.png)

**Comment by andyberry88** [comment]
The `<% .... %>` tokens in the page was caused by the index page being a `.html` rather than a `.jsp`. The bundle tokens not being replaced was fixed by #1457.
